python/pyspark/accumulators.py

```diff
@@ -123,14 +124,16 @@ def __reduce__(self):
     def value(self):
         """Get the accumulator's value; only usable in driver program"""
         if self._deserialized:
-            raise Exception("Accumulator.value cannot be accessed inside tasks")
```

Comment:
@LielBach, I am very sorry for late responses. I have thought about this and had some discussions offline with other people too. Can we, for now, switch `Exception` to use built-in exception classes for now? For example, `Exception` here could be just simply `RuntimeError`.

Throwing the plain `Exception` is the main reason I raised the JIRA in any event.